src/components/appMenuSections/dashboard/Dashboard.tsx

```diff
@@ -37,6 +47,37 @@ const Dashboard = () => {
     // TODO: change navigate with the corresponding join pool function as a normal user
   };
 
+  useEffect(() => {
+    const getCurrentNotifier = async () => {
+      const notifier = await readOnlyGetNotifier();
+      setCurrentNotifier(notifier);
+    };
+
+    getCurrentNotifier();
+  }, [currentNotifier]);
+
+  useEffect(() => {
+    if (userSession.isUserSignedIn()) {
+      const args = userSession.loadUserData().profile.stxAddress.testnet;
```

Comment:
declare at beginning networkUsed = mainnet if config-network is mainnet, else testnet ( for testnet and devnet ).

don't know if this is not fixed in a future pr. @BowTiedJohn is it?

Reply:
I don't think it is, but shouldn't we put it on Header Bar instead of Dashboard?

Reply:
it is not changed in a future opened pr. can be done in a future commit.